web_src/js/components/PullRequestMergeForm.vue

```diff
@@ -1,18 +1,18 @@
 <script lang="ts" setup>
-import {computed, onMounted, onUnmounted, ref, watch} from 'vue';
+import {computed, onMounted, onUnmounted, ref, shallowRef, watch} from 'vue';
 import {SvgIcon} from '../svg.ts';
 import {toggleElem} from '../utils/dom.ts';
 
 const {csrfToken, pageData} = window.config;
 
-const mergeForm = ref(pageData.pullRequestMergeForm);
+const mergeForm = shallowRef(pageData.pullRequestMergeForm);
```

Comment:
IIRC it doesn't need to be a ref, it should be read-only?

(It's fine to leave it if it is too difficult to change)